skymp5-server/cpp/server_guest_lib/MpObjectReference.cpp

```diff
@@ -979,11 +989,9 @@ void MpObjectReference::ProcessActivate(MpObjectReference& activationSource)
     auto refrRecord = espm::Convert<espm::REFR>(
       loader.GetBrowser().LookupById(GetFormId()).rec);
     auto teleport = refrRecord->GetData(compressedFieldsCache).teleport;
-    if (teleport) {
-      if (!IsOpen()) {
-        SetOpen(true);
-        RequestReloot();
-      }
+	// Worldspace sync fix here.
```

Comment:
There's no need to write that in the code itself. It's better to either leave comment on the PR (like this one)

Reply:
You're right, it was a reminder for myself. I will do that for further commits.